notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, the above should be possible using the \"open\" or \"import data\" button - the intention would be for it to all behave just as described above, but via a file browser (or command-line file path) instead of via a notebook method call."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multiple spectra in one `specviz` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because glupyter/glue support multiple data being loaded at the same time, it should also be possible to load data even if it's not all getting displayed initially.\n",
+    "\n",
+    "1.  In the notebook this is just a different method that otherwise behaves the same as `load_data`, just doesn't replace what's being shown - E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.add_data('some_other_spectrum.fits') #<- this would do exactly the same as `load_data` but just not show it\n",
+    "# OR \n",
+    "specviz.add_data('some_other_spectrum.fits', show=True)  #<- this would automatically add the flux as a new layer in the UI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, if the user does \"import data\" a second (or additional) time, `add_data` as described above can be used instead of `load_data`, as the user can always simply go into the viewer and change what's being shown via check boxes/drag-and-drop."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncertainties/mask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the `Spectrum1D` object loaded contains an uncertainty and/or mask attribute, these should be loaded ointo glue when `load_data` is called.  However, by default only the spectrum itself should be *shown* at that time. The user can show it later using standard `jdaviz` interactions if desired. We *may* wish some syntactic convenience for the notebook as well - e.g.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.show_uncertainty()\n",
+    "specviz.show_mask()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing the spectrum\n",
+    "\n",
+    "(only relevant for notebook/developer usage) - When a single spectrum is loaded using `load_data`, it should be easy to get that spectrum back again.  E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec = specviz.current_spectrum  #<- a Spectrum1D object\n",
+    "spec.flux, spec.uncertainty, ..."
+   ]
```

Comment:
All good points.  I agree `get_viewer_data` makes more sense as all spectra, and I like @havok2063's `visible_spectrum` to indicate that what I'm really thinking here is "what the user was most recently interacting with". (This might even raise an error if there's more than one showing since that's rather ambiguous. I'll try clarify that in the text.)  So I'll change all of the `current_spectrum` to `visible_spcetrum`, *but* note that that should be seen as interchangeable with `get_viewer_data()[0]` or similar depending on the situation.